### PR TITLE
Preparing tests to run a full tests set

### PR DESCRIPTION
**Pull request by vvarg229**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @vdomnich-yadro @dansingjulia @yadro-vavdeev
+*       oleg@nspcc.ru vvarg229@gmail.com
```
Comment (roman-khimov) [inline .github/CODEOWNERS:1]: Better have GH user names here. Add @roman-khimov and @cthulhu-rider as well.

**Review by roman-khimov — COMMENTED** [review]
>  testsuites: Rework skip and issues marks 

This one would be much better if it'd be a set of patches merged during the week, piece by piece and task by task (closing them one by one). Waiting till Friday to push the whole set at once is not a good style.

It's also a bit hard to digest "sanity" mark changes this way, but this may be a bit less of a problem, we'll see how PR/master runs differ and then tune it if needed.

**Comment by vvarg229** [comment]
> This one would be much better if it'd be a set of patches merged during the week, piece by piece and task by task (closing them one by one). Waiting till Friday to push the whole set at once is not a good style.

Before this path, the result of a full test run was a failed.
https://github.com/vvarg229/neofs-node/actions/runs/4874260200/jobs/8694959274
https://vvarg229.github.io/neofs-node/2/

Since these tests are not running right now, I think the main goal is to make consistently green tests.

> It's also a bit hard to digest "sanity" mark changes this way, but this may be a bit less of a problem, we'll see how PR/master runs differ and then tune it if needed.

I've already taken the time to "see how PR/master runs".
That's before "sanity" changed:
https://github.com/vvarg229/neofs-node/actions/runs/4882894675/jobs/8713531610
https://vvarg229.github.io/neofs-node/9/
Estimated time: 1h 13m 55s

And here is the result with this patch:
https://github.com/vvarg229/neofs-node/actions/runs/4883844610/jobs/8715750827
Estimated time: 31m 5s

**Comment by roman-khimov** [comment]
>  Estimated time: 31m 5s

Yeah, but I mean some tests are more important than others. But OK, we can fine-tune the list after the merge.

File: .github/CODEOWNERS (hunk 1 above)
Comment (vvarg229) [reply .github/CODEOWNERS]: Done

**Comment by vvarg229** [comment]
> > Estimated time: 31m 5s
> 
> Yeah, but I mean some tests are more important than others. But OK, we can fine-tune the list after the merge.

What was done before - was just that almost every test class was labeled as "sanity".
There was no selection of specific tests.

**Review by roman-khimov — APPROVED** [review]

**Comment by roman-khimov** [comment]
Please update relevant issues, btw. Many of them can be closed probably.

**Pull request merged**